src/components/about/PageTemplate.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components/macro'
+
+const PageTemplate = styled.article`
+  @media ${({ theme }) => theme.device.mobile} {
+    width: 80%;
+  }
+
+  max-width: 950px;
+  width: 66%;
+  overflow-y: scroll;
```

Comment:
We should let this component keep its height with `overflow-y: auto;` otherwise we get a weird "double scroll" behavior:

<img width="1039" alt="Screen Shot 2021-10-18 at 6 51 49 PM" src="https://user-images.githubusercontent.com/19193347/137821702-d9fcf8a7-5614-4bd7-88c7-895fe643e8c9.png">